### PR TITLE
Add docker image building workflow

### PR DESCRIPTION
**Pull request by yixinglu**
- Use github actions to build nebula service docker images and push them to docker hub at 2am every day night
- Fix docker-compose yaml and usage in readme
- Store nebula Dockerfiles in `docker` subdirectory rather than `vesoft-inc/nebula-docker-compose` repo

### CODE REVIEW

**Comment by yixinglu** [comment]
Jenkins go

**Comment by yixinglu** [comment]
Jenkins go

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-console:${{ github.sha }}
+          nightly_image_name: ${{ secrets.DOCKER_USERNAME }}/nebula-console:nightly
+        run: |
+          docker build -t ${IMAGE_NAME} -f docker/Dockerfile.console .
+          docker tag ${IMAGE_NAME} ${NIGHTLY_IMAGE_NAME}
+          docker push ${IMAGE_NAME}
+          docker push ${NIGHTLY_IMAGE_NAME}
+        shell: bash
+
+  nebula_graph:
```
Comment (laura-ding) [inline .github/workflows/docker.yaml]: It will test all? I can't find the `Dockerfile`

Comment (yixinglu) [reply .github/workflows/docker.yaml]: Dockerfiles in another PR #951

**Comment by yixinglu** [comment]
Jenkins go

**Comment by yixinglu** [comment]
Jenkins go

**Comment by yixinglu** [comment]
Jenkins go

**Comment by yixinglu** [comment]
Jenkins go

**Pull request merged**